pipelines/simple-notebook-pipeline/Simple Notebook Pipeline.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright 2019 Google Inc. All Rights Reserved.\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple notebook pipeline \n",
+    "\n",
+    "Welcome to your first steps with Kubeflow Pipelines (KFP). This notebook demos: \n",
+    "\n",
+    "* Defining a Kubeflow pipeline with the KFP SDK\n",
+    "* Creating an experiment and submitting pipelines to the KFP run time environment using the KFP SDK \n",
+    "\n",
+    "Reference documentation: \n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/sdk-overview/\n",
+    "* https://www.kubeflow.org/docs/pipelines/sdk/build-component/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prerequisites: Install the pipelines SDK\n",
+    "If you followed the [notebook setup instructions](https://www.kubeflow.org/docs/notebooks/setup/) then you don't have to run the next cell.  You only need to install the KFP package once in your notebook server.  You can find the latest KFP package [here](https://github.com/kubeflow/pipelines/releases)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# If you've already done his during setup you don't need to do it again\n",
+    "KFP_PACKAGE = 'https://storage.googleapis.com/ml-pipeline/release/0.1.23/kfp.tar.gz'\n",
+    "!pip3 install $KFP_PACKAGE --upgrade"
```

Comment:
For some reason this installs kfp v0.1.22. How about just using
`!pip install kfp --upgrade`?

Since the pipeline is simple, I wouldn't expect many breaking changes with version updates.

Reply:
Done.